### DISCUSSION TITLE
Formily视频教程话题收集

### DISCUSSION THREAD

**Discussion by janryWang** [alibaba/formily#2502] [Show and tell]
后面会陆续出一些视频教程，但是需要收集一些话题，大家有想法的，评论一下就行了，注意，视频教程不会讲文档已有的内容，只讲文档也说不清楚，需要实操才能理解的干货

**Comment by sage-wen** [comment]
希望讲一下关于自增列表的一些难点，例如：
1.自增表格如何实现表头列和数据列的联动
2.如何保障列表海量数据的渲染、联动、校验、字段查找等的性能
3.嵌套子列表的性能

**Comment by iscmcloud** [comment]
希望讲一下自定控件的相关内容，自定义组件的基本流程和写法

**Reply by sarah0730** [reply]
1、希望讲解下写自定义组件的基本写法
2、想了解designable/formily/antd/和 formily/antd 的区别？(设计器里面自定义了组件是否在渲染formily的时候也要自定义组件)

**Comment by liby2006** [comment]
希望讲一下关于自增列表 的自定义操作，
如果通过右侧的属性配置 去添加操作 ，如何优雅的给当前的组件ArrayTable.Column 增加 目标操作的 children TreeNode 
右侧属性配置 添加自定义操作，如果不合适，应该使用什么方式呢

**Comment by chensce** [comment]
<img width="254" alt="Screen Shot 2021-12-07 at 3 11 13 PM" src="https://user-images.githubusercontent.com/93559562/144983106-aee24466-7921-4ac9-b846-431838936664.png">
建议把这几种场景，补充完整，文档，和视频，像第一个注册表单就很好，来源于实际业务，解决方案也是很专业。
其它场景，也按照注册那个，来编写。
最好做成playground那样的。
读者，还能自己去动手，体会体会。
